src/CMakeLists.txt

```diff
@@ -134,3 +134,13 @@ install(
     ARCHIVE DESTINATION lib/static COMPONENT applications
     INCLUDES DESTINATION include COMPONENT applications
 )
+
+# Configure appimagetool.AppImage component
+install(
+    TARGETS appimagetool AppRun
+    RUNTIME DESTINATION bin COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL
+)
+
+install(FILES "${PROJECT_SOURCE_DIR}/resources/appimagetool.desktop" DESTINATION share/applications COMPONENT appimagetool.AppImage EXCLUDE_FROM_ALL)
```

Comment:
It should be safe to install these dependencies in the `appimagetool` components, shouldn't it? Otherwise, I'd say we can merge this.

Reply:
Safe, it's. But I don't see any use for it. 

Or you just mean changing the component name from appimagetoo.AppImage to just appimagetool ?

Reply:
Yeah, I was thinking about renaming the component, as the installation is quite generic, and will work for non-AppImage related purposes as well.

Reply:
Done.